code/src/App.js

```diff
@@ -1,9 +1,22 @@
 import React from 'react'
+import { Provider } from 'react-redux'
+import { combineReducers, configureStore } from '@reduxjs/toolkit'
+import tasks from 'reducers/tasks'
+import OverView from 'components/OverView'
+
+const reducer = combineReducers({
+  tasks: tasks.reducer,
+})
+
+const store = configureStore({
+  reducer,
+})
+
 
```

Comment:
That's it! After you've done the above steps, your to-do list will save the state locally in the browser, and if you leave the page or refresh it, they'll still stay there 💖